pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1008,7 +998,6 @@
       "AssemblyVersionInPackageVersion": {
         "4.0.0.0": "4.0.0",
         "4.0.10.0": "4.0.10",
-        "4.1.0.0": "4.0.12",
         "4.0.11.0": "4.0.11"
```

Comment:
This was inconsistent with the actual repo state.  Assembly version 4.1.0.0 does not currently build, even if it did it should never be put in a package versioned 4.0.12.